packages/shared/src/components/modals/SquadsBetaModal.tsx

```diff
@@ -1,9 +1,10 @@
-import React, { ReactElement, useState } from 'react';
+import React, { ReactElement } from 'react';
+import { usePrompt } from '../../hooks/usePrompt';
 import SquadsBackgroundSvg from '../../svg/SquadsBackground';
 import { Button } from '../buttons/Button';
 import { Justify } from '../utilities';
 import { Modal } from './common/Modal';
-import { SquadsConfirmation } from './SquadsConfirmationModal';
+import { quitSqualModal } from './NewSquadModal';
```

Comment:
`quitSquadModal`

Reply:
Maybe this should also be in utils rather?
As it's not bound to one component

Reply:
I agree this should be moved. I was having some dependency cycle issues earlier. But I have now found a common file that doesn't produce the same error. Thanks for the review!